obspy/io/rg16/__init__.py

```diff
@@ -1,20 +1,88 @@
 """
-Functions to read waveform data from Receiver Gather 1.6-1 foramt.
-This format is used for continuous data by Farfield's (fairfieldnodal.com)
-Zland product line (http://fairfieldnodal.com/equipment/zland).
+obspy.io.rg16 - receiver gather v1.6 read support for ObsPy
+=============================================================
```

Comment:
Same length as the text please, sphinx will show a warning otherwise (adding on top of hundreds or thousands of warnings currently.. ;-)